### DISCUSSION TITLE
Xbox 360 like launcher for android/pc [suggestion]

### DISCUSSION THREAD

**Discussion by Bruhmigo** [EmiyaSyahriel/CrossLauncher#25] [Ideas]
Read the title

**Comment by EmiyaSyahriel** [comment]
Weren't there one already? I found some tutorial [here at XDA-Developers](https://forum.xda-developers.com/t/how-to-xbox-dashboard-style-homescreen-tutorial.1590019/) using a non X360 dedicated launcher. as for PC, IDK.

Aside from that, I don't know if I'm able to incorporate X360's Home Screen (Menu? Dashboard?) into to this very launcher since the XMB Styles were all shares same code as most have same look and functionality. 

I might create a new repository for X360's if you can give me some reference, but I can't promise when will I start the project and when will it stable enough for release, or even would I really start it or not. and making one for PC (Windows specifically) is a bit too complicated, so I will skip it probably.

And If you look around this repository, I'm not even that active updating lately.

Also, for my entire life, I haven't seen the real X360 Home Screen, it's variants (like PS3/PSP XMB Screens), how it's behaviour and how it should feel when you navigate through it's menus, as I never own one before.

**Comment by Bruhmigo** [comment]
Well if you have time ill refrence this dashboard ill love to see it on android https://youtu.be/olTeUQbIjoU

**Comment by EmiyaSyahriel** [comment]
Anyway, I'll move this issue to a discussion instead since this is not an issue regarding this app.

**Reply by trikx2** [reply]
But there is a program XMBPC https://www.youtube.com/watch?v=uLuF5xB6nTI&t=350s
But the program does not start the game, it's a pity
